app-editors/atom/atom-1.11.2.ebuild

```diff
@@ -0,0 +1,398 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit flag-o-matic python-any-r1 eutils multiprocessing rpm
```

Comment:
Ah, I see.  Didn't realise eapply was part of the EAPI now.  Thanks for pointing this out. I've tested and pushed the fix.